bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -114,6 +114,7 @@ public class ServerConfiguration extends AbstractConfiguration<ServerConfigurati
     protected static final String GC_OVERREPLICATED_LEDGER_WAIT_TIME = "gcOverreplicatedLedgerWaitTime";
     protected static final String GC_OVERREPLICATED_LEDGER_MAX_CONCURRENT_REQUESTS =
             "gcOverreplicatedLedgerMaxConcurrentRequests";
+    protected static final String GC_ZK_OP_RATE_LIMIT = "gcZkOpRateLimit";
```

Comment:
Have you considered using `gcMetadataOpRateLimit` instead? The underlying metadata store could be ZooKeeper, etcd, or something else, so using a more general name rather than one tied to a specific implementation is better.